src/Essential.Diagnostics.SeqTraceListener/Diagnostics/SeqTraceListener.cs

```diff
@@ -164,6 +167,43 @@ public int BatchSize
             }
         }
 
+        /// <summary>
+        /// Gets or sets the error handling of sync batches. Use true to catch each exception. Default is false.
+        /// </summary>
+        public bool SyncErrorHandling
```

Comment:
Use the name "IndividualSendIgnoreErrors".

Use the summary: "Gets or sets a flag whether to ignore errors when using individual send mode (batch size is 0). Use true to catch and ignore all exceptions when sending. Default is false. It has no effect if batch size is > 0."

Reason: 
* Sync is an abbreviation and should not be used.
* My first interpretation was "Synchronize error handling"
* You probably meant "Synchronous error handling"
* However, synchronous is only mentioned in the internal comments for Enqueue, and even then only in brackets. The name used in the consumer accessible public doc comments is "individual send", in the context of the "BatchSize" property.
* The name and description should link it back to this other property, i.e. make it clear when it is relevant.
* The code is not actually handling the error, but ignoring (suppressing) it; "ignore errors" is the more common term.
* Handling would be, for example, the retry mechanism in the asynchronous send.

Reply:
On this you are completely correct.